bitmap-team-awesome/test/bit-file-helper-test.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const bitFileHelper = require('../lib/bit-file-helper');
+const expect = require('chai').expect;
+
+describe('module bitFileHelper', function(){
+  describe('testing bitFileHelper() read file', function(){
+    it('should return an error', function(){
+      bitFileHelper(`${__dirname}/../../assets/notBM.bmp`, function(err){
+        expect(!!err).to.equal(true);
```

Comment:
@meeksfred good error catching test - this overall module test, for your bitmapFileHelper module, could include additional test components - these individual components are all up to you to determine but checking for the appropriate data structure (which you did a bit of below by looking for the colorOffSet property) and file type are good starting points.